### ISSUE TITLE
No Volume Control

### ISSUE THREAD

**Issue by Krutonium** [IronManMark20/ShellSharp#18]
When explorer is shut down, and only Shell Sharp is running, I have no volume control. We should also have a volume control.

**Comment by Krutonium** [comment]
http://stackoverflow.com/questions/13139181/how-to-programmatically-set-the-system-volume/13139478#13139478

**Comment by Krutonium** [comment]
Should also respond to keyboard volume keys.

**Comment by emmatyping** [comment]
Yep. I'll be adding a keyboard hook for Win+E, Win+D, and Alt+Tab, so I'll throw the keyboard stuff  in when I do.
